FirstProject/Program.cs

```diff
@@ -1 +1,40 @@
-﻿
\ No newline at end of file
+﻿using FirstProject;
+using System.ComponentModel.Design;
+
+Console.WriteLine("FirstProject");
+
+
+    BasedCalculator basedcalculator = new BasedCalculator();
+Console.WriteLine("Kalkulator podstawowy");
```

Comment:
Raczej to  powinno się wydzielić do oosbnej klasy obsługującej wyświetlanie sie menu i tekstów czy coś takeigo. Na potrzeby tego taska nei jest to potrzebne wiec mozna nawet usunąć